graylog2-server/src/main/java/org/graylog/plugins/pipelineprocessor/rulebuilder/RuleBuilderRegistry.java

```diff
@@ -40,7 +39,7 @@ public RuleBuilderRegistry(FunctionRegistry functionRegistry,
     }
 
     public Map<String, RuleFragment> conditions() {
-        final Stream<RuleFragment> functionConditions = functionRegistry.all()
+        final Stream<RuleFragment> functionConditions = functionRegistry.allWithInternal()
```

Comment:
@bernd I think we should flag this so that we don't return internal actions & conditions  in the resources, otherwise, they will be available in the rule builder UI

Reply:
@kodjo-anipah Good catch, thank you! We need the function for the generator, but it shouldn't be exposed. So we have to create two methods that we use in different places?

Reply:
Yes that works perfectly